src/DRAMPower/DRAMPower/standards/ddr5/interface_calculation_DDR5.h

```diff
@@ -12,12 +12,11 @@ class DDR5;
 
 class InterfaceCalculation_DDR5 {
    public:
-    InterfaceCalculation_DDR5(DDR5 &ddr);
+    InterfaceCalculation_DDR5(const MemSpecDDR5 &memspec);
 
```

Comment:
This API change reinforces separation of concerns and removes the circular dependency between DDR5 and InterfaceCalculation_DDR5